examples/pubsub-demo/build.gradle

```diff
@@ -48,4 +50,17 @@ dependencies {
 // Don't need groovydoc for example project
 tasks.named('groovydoc') {
     enabled = false
-}
\ No newline at end of file
+}
+
+bootJar {
+    duplicatesStrategy = DuplicatesStrategy.INCLUDE
```

Comment:
why?

Reply:
To work around duplicates that exist during the build.  These were new after upgrade to gradle 8.10, but DuplicatesStrategy is set in other Grails projects.